source/user-manual/configuring-cluster/advanced-settings.rst

```diff
@@ -0,0 +1,146 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+Agents connections
+======================
+
+    In this section we are going to explore other ways to configure our cluster. We'll see that besides the basic way described in our :ref:`Getting started <gt-cluster>`, we have the options to use a load balancer or a failover mode.
+
+    .. important::
+
+        We recommend using the **load balancer** option. This way the agents will be able to report to the nodes in a distributed way and it will be the balancer the one who assigns to which worker they report, this way the load will be more distributed and in case of a fall in some worker node, their agents will **reconnect** to another one.
```

Comment:
```suggestion
        We recommend using the **load balancer** option. This way the agents will be able to report to the nodes in a distributed way and it will be the load balancer who assigns which worker they report to. Using this option we can better distribute the load and in case of a fall in some worker node, its agents will **reconnect** to another one.
```